packages/@textlint/ast-node-types/src/NodeType.ts

```diff
@@ -54,8 +54,6 @@ export interface TxtNode {
     loc: TxtNodeLocation;
     // `parent` is created by runtime
     parent?: TxtParentNode;
-
-    [index: string]: any;
```

Comment:
Remove `[index:string]: any;` from `TxtNode`

Previoulsy, `Omit<TxtNode, "type">` return any objects. It is bad.